src/server/scripts/EasternKingdoms/Karazhan/boss_nightbane.cpp

```diff
@@ -145,6 +146,10 @@ struct boss_nightbane : public BossAI
         {
             DoCastRandomTarget(SPELL_SEARING_CINDERS);
             context.Repeat(10s);
+        }).Schedule(1500ms, GROUP_GROUND, [this](TaskContext context)
+        {
+            DoCastRandomTarget(SPELL_CLEAVE);
```

Comment:
Should this `DoCastRandomTarget` not be `DoCastVictim`? Other instances of cleave spells (like [Magtheridon](https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/scripts/Outland/HellfireCitadel/MagtheridonsLair/boss_magtheridon.cpp#L165)) do use `DoCastVictim`.

Reply:
Yep